services/QuillLMS/spec/controllers/topics_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TopicsController do
+  let!(:topic) { create(:topic) }
+  let!(:level_two_topic) { create(:topic, level: 2) }
+  let!(:level_three_topic) { create(:topic, level: 3) }
+
+  describe '#index' do
+    it 'should return all the topics' do
+      get :index
+      parsed_response = JSON.parse(response.body)
+      topic_results = parsed_response["topics"]
+      Topic.all.each do |t|
```

Comment:
Since you're only comparing ids, you could rewrite lines 15-17 as `expect(topic_results.pluck("id")).to match_array Topic.pluck(:id)`